tests/wpt/metadata/webvtt/api/VTTCue/align.html.ini

```diff
@@ -0,0 +1,5 @@
+[align.html]
```

Comment:
We should add a `__dir__.ini` in tests/wpt/metadata/webvtt like https://github.com/szeged/servo/blob/mmatyas__webgl_fns_framebuf_invalidate/tests/wpt/webgl/meta/conformance2/__dir__.ini that sets the new preference, and update the test results.

Reply:
I only updated the test results expectations for `webvtt/api`. Should I do it for all 317 tests in webvtt?

Reply:
I recommend adding `[webvtt]` and `[api]` entries to https://github.com/servo/servo/blob/master/tests/wpt/include.ini so we only run the tests that are most relevant while we're in the early stages.

Reply:
Does that .ini file has any effect if I

Reply:
Does the .ini file has any effect if I run the tests with a path? There still seems to be the same number of tests (317) that are being run.
`$ ./mach test-wpt tests/wpt/web-platform-tests/webvtt/`
```
@@ -175,6 +175,10 @@ skip: true
   skip: false
 [webvr]
   skip: false
+[webvtt]
+  skip: true
+  [api]
+    skip: false
 [webxr]
   skip: false
 [WebIDL]

```

Reply:
Running with a path ignores the contents of include.ini.

Reply:
I hope I got it right. This is all magic for me now.